sonar-db/src/test/java/org/sonar/db/DbTester.java

```diff
@@ -97,13 +103,45 @@ public static DbTester create(System2 system2) {
   public static DbTester createForSchema(System2 system2, Class testClass, String filename) {
     String path = StringUtils.replaceChars(testClass.getCanonicalName(), '.', '/');
     String schemaPath = path + "/" + filename;
-    return new DbTester(system2, schemaPath);
+    return new DbTester(system2, schemaPath).setDisableDefaultOrganization(true);
+  }
+
+  private void initDbClient() {
+    TransientPicoContainer ioc = new TransientPicoContainer();
+    ioc.addComponent(db.getMyBatis());
+    ioc.addComponent(system2);
+    ioc.addComponent(new SequenceUuidFactory());
+    for (Class daoClass : DaoModule.classes()) {
+      ioc.addComponent(daoClass);
+    }
+    List<Dao> daos = ioc.getComponents(Dao.class);
```

Comment:
My point was only that if start() isn't called and some classes are startable, they won't be started and some initialization might be missing. Lucky, in case of DAOs, it is not necessary.